# For sumulation world
# Map between location names and coordinates
location_names:
  - {name: "visiting room",  x: 0.5,   y: 2.8,     frame: "map"}
  - {name: "hall",           x: 1.3,   y: 1.0,     frame: "map"}
  - {name: "dinning room",   x: -0.3,  y: -0.5,   frame: "map"}
  - {name: "your base",      x: 0.0,   y: 0.0,     frame: "map"}

  - {name: "front",  x: 1.0,  y: 0.0,   frame: "base_footprint"}
  - {name: "back",  x: -1.0,  y: 0.0,   frame: "base_footprint"}
  - {name: "left",  x: 0.0,  y: 1.0,   frame: "base_footprint"}
  - {name: "right",  x: 0.0,  y: -1.0,   frame: "base_footprint"}


# List of locations to be visited when patrolling
patrolling_itinerary:
  - "visiting room"
  - "your base"


